src/pages/index.tsx

```diff
@@ -35,12 +35,12 @@ const IndexPage: React.FC = () => (
     <PageLinks />
     <GeneralBodyContainer>
       <GeneralDescription
-        title="Board Applications Open!"
+        title="Welcome to VandyHacks!"
         text1={
-          `Welcome to VandyHacks! Our 2021/VH VIII Board applications are currently open until 11:59 PM CST on January 20th, 2021.`
+          `Our 2021/VH VIII Board applications have closed, and we are ecstatic about all our new members!`
```

Comment:
```suggestion
          `Our 2021/VH VIII Board applications have closed, and we welcome all our new members!`
```
ecstatic feels too cool here

Reply:
are we not cool?

Reply:
I feel like "we welcome all our new members" sounds a bit off. How about split the difference and be like we're excited to welcome all our new members?

Reply:
okay sounds good let's do that

Reply:
ecstatic just feels out of place-ish, like it draws attention to itself